### ISSUE TITLE
Snap package build for Linux. For an easy one click install from Snap Store in Ubuntu. 

### ISSUE THREAD

**Issue by heidiwenger** [streamlink/streamlink-twitch-gui#834]
### Checklist

- [X] I understand that **Streamlink Twitch GUI** is just a launcher for **Streamlink**
- [X] [I have read the contribution guidelines](https://github.com/streamlink/streamlink-twitch-gui/blob/master/CONTRIBUTING.md)
- [X] [I have checked the list of open and recently closed issues](https://github.com/streamlink/streamlink-twitch-gui/issues?q=is%3Aissue+sort%3Aupdated-desc)
- [X] [I have checked the commit log of the master branch](https://github.com/streamlink/streamlink-twitch-gui/commits/master)

### Description

Ubuntu is very popular, especially with new users. Well, i was trying to explain in a hurry on the phone how to make an AppImage executable. Only that extra effort, instead of just clicking "install" in the Software center, was too much for a commoner. 

So i thought to suggest here whether a snap package for Streamlink-Twitch-GUI would be possible? Snaps are also sandboxed by default.

There is no other viable Twitch "client" for Linux currently. 

Thanks.

**Comment by heidiwenger** [comment]
Well, this is Stratovarius' progressive side masterpiece that i know! Made even without the maestro Timo Tolkki.

https://www.youtube.com/watch?v=ikDS1TBAElo

**Comment by heidiwenger** [comment]
> I was talking about fixing the issues and implementing specific flatpak workarounds into the applications, because this is a requirement for circumventing the sandboxing.
> 
> To be honest, I am ok with "occasional and regular PC users" having to figure out how to make appimages executable. This is documented everywhere, in Streamlink's docs and in Streamlink Twitch GUI's wiki. If people don't want to read and learn, then this is not my problem. There are also tools for this which do this automatically, which is also documented.
> 
> As said, I'm not interested in snaps. And flatpaks are currently not worth the effort, so I'm going to close the thread... Not sure why you've posted this youtube link...

Yea i was posting a video in other place, it was an accident :D

**Comment by parphelion** [comment]
Now that ffmpeg is available in flatpak's runtime, would it be possible for the application to be packaged as a flatpak?

**Comment by bastimeyer** [comment]
> would it be possible for the application to be packaged as a flatpak?

No. I have already answered this in detail in my comment above. Read it.
https://github.com/streamlink/streamlink-twitch-gui/issues/834#issuecomment-945017671

Unless someone submits pull requests for both Streamlink and Streamlink Twitch GUI which work around the sandboxing (as described above), this won't happen. I am not interested in spending any time on implementing this myself, and I am not interested in discussing this further.

There are appimages available if your distro's native package manager doesn't include (recent) packages for Streamlink and/or Streamlink Twitch GUI, so use these.

And if one wants to build an "opinionated flatpak package" for Streamlink Twitch GUI that bundles
1. Python + dependencies
2. Streamlink + dependencies
3. Streamlink Twitch GUI + dependencies
4. MPV or VLC + dependencies
so that the sandboxing doesn't need to get worked around, then they can do that themselves.
